libs/shared/form-fields/src/lib/CheckboxController/CheckboxController.spec.tsx

```diff
@@ -47,13 +47,35 @@ describe('CheckboxController', () => {
     ]
     const { getByText } = render(
       <Wrapper>
-        <CheckboxController
-          id="values"
-          error="error indeed"
-          options={options}
-        />
+        <CheckboxController id="values" options={options} />
       </Wrapper>,
     )
     expect(getByText('nice tooltip')).toBeInTheDocument()
   })
+
+  it('should render a sublabel below an option if wished and when large set to true', () => {
+    const options = [
+      { label: 'some checkbox', value: 'off', subLabel: 'nice sublabel' },
+      { label: 'another checkbox', value: 'on' },
+    ]
+    const { getByText } = render(
```

Comment:
Please use screen.getByText instead of this. (source https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#not-using-screen)